front-end/src/components/FriendRequestButton.tsx

```diff
@@ -11,21 +11,36 @@ interface Props {
 }
 
 export default function FollowRequestButton(props: Props) {
-  const url = process.env.REACT_APP_API_URL + "/api/author/" + props.currentAuthor?.id + "/followers/" + props.loggedInUser?.authorId + "/";
+  // TODO: change this to use currentAuthor.url
+  const authorUrl = process.env.REACT_APP_API_URL + "/api/author/" + props.currentAuthor?.id + "/followers/" + props.loggedInUser?.authorId + "/";
+  const inboxUrl = process.env.REACT_APP_API_URL + "/api/author/" + props.currentAuthor?.id + "/inbox/";
   const loggedInUserUrl = process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId;
 
   const sendFollowRequest = () => {
     // TODO: add authentication
     // get the loggedinuser author object
     axios.get(loggedInUserUrl).then(res => {
       if (!props.isFollower) {
-        axios.put(url, res.data).then(res => {
+        axios.put(authorUrl, res.data).then(res => {
           if (res.status === 201) {
             props.setIsFollower(true);
           }
         });
+
+        let follow = {
+          type: "follow",
+          summary: "", // do we need to make custom messages when we follow? prob not
+          actor: res.data,
+          object: props.currentAuthor
+        };
+
+        axios.post(inboxUrl, follow).then(res => {
+          if (res.status === 201) {
+            console.log("friend request sent");
+          }
+        });
```

Comment:
This should be moved to the above if statement so that it sends the follow message to the inbox only when we get a 201

Reply:
done